starboard/build/config/BUILDCONFIG.gn

```diff
@@ -359,11 +359,11 @@ template("evergreen_loader") {
     "//starboard/loader_app:copy_loader_app_content",
   ]
   if (cobalt_font_package == "empty") {
-    shared_data_deps += [ "//cobalt/content/fonts:copy_font_data" ]
+    shared_data_deps += [ "//staboard/content/fonts:copy_font_data" ]
```

Comment:
`staboard` -> `starboard`

Reply:
Oops, thanks for catching ! This would have tripped up at least a few partners